pkg/clusterspec/clusterspec.go

```diff
@@ -10,34 +10,45 @@ import (
 )
 
 type ClusterSpec struct {
-	Name              string
-	ApiProvider       string
-	CloudProvider     string
-	Type              string
-	KubernetesVersion string
-	NodeType          string
-	NodeCount         int
-	MasterNodeCount   int
-	Region            string
-	PodCidrBlock      string
-	SshKeyName        string
-	Tags              string
-	Description       string
+	Name                      string
+	ApiProvider               string
+	CloudProvider             string
+	Type                      string
+	KubernetesVersion         string
+	NodeType                  string
+	NodeCount                 int
+	MasterNodeCount           int
+	Region                    string
+	PodCidrBlock              string
+	SshKeyName                string
+	Tags                      string
+	Description               string
+	ClusterAutoscalerEnabled  bool
+	ClusterAutoscalerMinNodes int
+	ClusterAutoscalerMaxNodes int
```

Comment:
Just curious in general, it seems like a mix of variables in this type—vaguely similar to the qbert API. Will this be the final API?